packages/flags/src/flags/buildInfo.ts

```diff
@@ -28,7 +28,9 @@ export async function getBuildInfo(): Promise<BuildInfoFile> {
         }
     }
 }
-export let env: BuildInfoFile
+export let env: BuildInfoFile = {
```

Comment:
access to this variable before calling setupBuildInfo should be a fatal error, so I'd rather leave it `undefined`.